packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -172,6 +196,12 @@ const GeneralTab = decorate([
                 })}
               </div>
             )}
+            {vmCreator !== undefined && <p className='mb-0'>{_('createdBy', { user: vmCreator.email })}</p>}
+            {vmTemplate !== undefined && (
+              <p className='mb-0'>
+                {_('originalTemplate')} {vmTemplate.name_label}
+              </p>
+            )}
```

Comment:
In order to always help with discoverability, what do you think about not hiding the messages when the values are undefined but rather show something like "Created by unknown user"? Or would it be too much/too confusing?

And also, if we do that, we could have a single message that would look like `"Created by {user} on {date} with template {template}"`.

Reply:
I don't think it's a good idea knowing that `vmCreator` will be `undefined` for all non-admin users.
BTW, I think showing "unknown" if user is admin and `vmCreator` or `vmTemplate` is `undefined` is good intermediary.

Reply:
I think "unknown" for the user should be shown when we don't have the info, not when the user isn't allowed to see it.